src/gg.regression.unity.bots/Samples~/ThirdPersonDemoURP/Bots/QuickstartBot/BotEntryPoint.cs

```diff
@@ -60,11 +61,11 @@ public override void ProcessTick(RG rgObject)
                 _destination = new Vector3(x, 0, z);
                 
                 // Also, shoot the enemy while we are waiting
-                var enemies = rgObject.FindEntities("Enemy");
+                var enemies = rgObject.FindEntitiesWithType<RGStateEntity_EnemyController>();
                 if (enemies.Count > 0)
                 {
-                    rgObject.PerformAction(new RGActionRequest_MoveInDirection(new Vector2(0, 0)));
-                    rgObject.PerformAction(new RGActionRequest_SelectAndAttackEnemy(enemies[0].id, 0));
+                    rgObject.PerformAction(new RGActionRequest_PlayerInputControl_MoveInDirection(new Vector2(0, 0)));
```

Comment:
How do we feel about the longer names in bot code? Suppose it's fine and more precise maybe.

Reply:
I don't necessarily love the long names, but it kind of is what it is...

The classnames correlate to the behaviours they go with.
1. For sanity sake so you can easily see what was generated from what
2. You don't have conflicts if the same action name is used in different behaviours